### PR TITLE
Including power in beaconData cause malformed URL since builder.setInclu...

### PR DESCRIPTION
**Pull request by alexdennis**
...deTxPowerLevel(false);

### CODE REVIEW

**Comment by ryanbateman** [comment]
I believe my [previous pull request](https://github.com/google/uribeacon/pull/59) tried to handle the same issue but it was resolved as an old release that no longer reflects fixed code.

**Comment by schilit** [comment]
I think the problem might be that the 16-bit UUID (FE D8) is included by Android by the addServiceData() so the first 2 bytes are duplicated. Try removing them and seeing how it works.

**Comment by alexdennis** [comment]
I tested the lastest master however the malformed url still existed. So in my branch, I added back the flags and Adv Tx Power and removed the repeated UUID and its working for me.

File: beacons/android/app/src/main/java/org/uribeacon/example/beacon/UriBeaconAdvertiserActivity.java [hunk 1]
```diff
@@ -55,16 +56,14 @@ private AdvertiseData getAdvertisementData() {
...
-        beaconData[2] = 0x00; // flags
-        beaconData[3] = 0x20; // transmit power
-        beaconData[4] = 0x00; // http://www.
-        beaconData[5] = 0x65; // e
-        beaconData[6] = 0x66; // f
-        beaconData[7] = 0x66; // f
-        beaconData[8] = 0x08; // .org
+        byte[] beaconData = new byte[7];
+        beaconData[0] = 0x00; // flags
+        beaconData[1] = 0x20; // transmit power
```
Comment (g-ortuno) [inline beacons/android/app/src/main/java/org/uribeacon/example/beacon/UriBeaconAdvertiserActivity.java]: Would you mind changing this to:
```java
beaconData[1] = (byte) 0xBA;
```

Which seems to be the correct Tx Power for the nexus 6.

Comment (alexdennis) [inline beacons/android/app/src/main/java/org/uribeacon/example/beacon/UriBeaconAdvertiserActivity.java]: Certainly, doing now.

**Comment by g-ortuno** [comment]
LGTM

**Comment by schilit** [comment]
Thanks! Glad you persevered and got this working.

**Pull request merged**